src/tuner/nccl_ofi_tuner.c

```diff
@@ -48,13 +47,16 @@ ncclResult_t nccl_ofi_tuner_init(size_t nRanks, size_t nNodes, ncclDebugLogger_t
 	return ncclSuccess;
 }
 
+// ReSharper disable once CppDFAConstantFunctionResult
```

Comment:
Can these annotations be a separate dot config instead?

Reply:
I'll just drop them.